docs/effects.md

```diff
@@ -1,4 +1,20 @@
-### MoveEffect
+# Effects
+An effect can be applied to any `PositionComponent`, there are currently two effects that you can use and that is the MoveEffect and the ScaleEffect.
+
+If you want to create an effect that only runs once, specify the three first parameters Size/Position, Speed and Curve.
```

Comment:
I would say requere parameters instead of size, position ad etc, as in the future we can have different effects other than position effects